components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLAnalysisHandler.java

```diff
@@ -22,12 +22,7 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.ml.commons.constants.MLConstants;
-import org.wso2.carbon.ml.commons.domain.FeatureSummary;
-import org.wso2.carbon.ml.commons.domain.MLAnalysis;
-import org.wso2.carbon.ml.commons.domain.MLCustomizedFeature;
-import org.wso2.carbon.ml.commons.domain.MLHyperParameter;
-import org.wso2.carbon.ml.commons.domain.MLModelConfiguration;
-import org.wso2.carbon.ml.commons.domain.MLModelData;
+import org.wso2.carbon.ml.commons.domain.*;
```

Comment:
Importing class itself is preferred for code clarity purposes. Otherwise it'd be hard to find from which package a class comes.